### ISSUE TITLE
Movie Recording/Playing trouble with Contra ROM.

### ISSUE THREAD

**Issue by achpile** [0ldsk00l/nestopia#150]
I got this bug only with Contra ROM (tested on J and U versions). I don't know, what's wrong... Recording or Playing a movie. Tested on Linux 1.46 and latest github build (Ubuntu 15.04 32bit and 64bit) and 1.46 win32. 

1.40 and 1.36 works fine.

I don't know how to describe the trouble... On movie just everything going wrong (After starting game).

**Comment by rdanbrook** [comment]
Try it with SMB3 or Castlevania 3 and let me know if you have similar problems.

**Comment by achpile** [comment]
Yeah! Absolutely same thing! Problems with mapper?

**Comment by rdanbrook** [comment]
I have noticed problems in these games when using the rewinder, and other MMC3/6 games. Depending on what region the ROM is, the mapper for Contra will be different. I have noticed the problem in Contra with the rewinder before, but it's not an MMC3 or 6 game.

To be honest I don't really know how to fix this issue, I will probably have to dig really deep to get to the bottom of it. It appears the rewinder issue is related though.
